rollup.config.mjs

```diff
@@ -10,46 +10,46 @@ import postcss from 'rollup-plugin-postcss';
 import { ROLLUP_INCLUDE_USE_CLIENT } from './rollup-exclude-use-client.mjs';
 
 export default [
-  {
-    input: 'src/index.ts',
-    output: [
-      {
-        file: 'dist/cjs/index.js',
-        format: 'cjs',
-        sourcemap: true,
-      },
-      {
-        file: 'dist/esm/index.js',
-        format: 'esm',
-        sourcemap: true,
-      },
-    ],
-    plugins: [
-      peerDepsExternal(),
-      nodeResolve(),
-      commonjs(),
-      typescript({
-        exclude: ['src/app.js', 'src/app.css', 'src/container.js', 'src/index.js'],
-      }),
-      postcss({
-        modules: true,
-        extract: true,
-      }),
-      banner((chunk) => {
-        if (!ROLLUP_INCLUDE_USE_CLIENT.includes(chunk.fileName)) {
-          return "'use client';\n";
-        }
+  // {
+  //   input: 'src/index.ts',
+  //   output: [
+  //     {
+  //       file: 'dist/cjs/index.js',
+  //       format: 'cjs',
+  //       sourcemap: true,
+  //     },
+  //     {
+  //       file: 'dist/esm/index.js',
+  //       format: 'esm',
+  //       sourcemap: true,
+  //     },
+  //   ],
+  //   plugins: [
+  //     peerDepsExternal(),
+  //     nodeResolve(),
+  //     commonjs(),
+  //     typescript({
+  //       exclude: ['src/app.js', 'src/app.css', 'src/container.js', 'src/index.js'],
+  //     }),
+  //     postcss({
+  //       modules: true,
+  //       extract: true,
+  //     }),
+  //     banner((chunk) => {
+  //       if (!ROLLUP_INCLUDE_USE_CLIENT.includes(chunk.fileName)) {
+  //         return "'use client';\n";
+  //       }
 
-        return undefined;
-      }),
-    ],
-  },
-  {
-    input: 'dist/esm/index.d.ts',
-    output: [{ file: 'dist/index.d.ts', format: 'esm' }],
-    plugins: [dts()],
-    external: [/\.(css|less|scss)$/],
-  },
+  //       return undefined;
+  //     }),
+  //   ],
+  // },
+  // {
+  //   input: 'dist/esm/index.d.ts',
+  //   output: [{ file: 'dist/index.d.ts', format: 'esm' }],
+  //   plugins: [dts()],
+  //   external: [/\.(css|less|scss)$/],
+  // },
```

Comment:
Hey @NicolaasDieBaas :) We're going to need the uncommented when we commit to the repo. We only comment it out during development so the `build:watch` command works smoothly.